website/releases/release-0.11.0.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Release 0.11.0"
+sidebar_position: 2
+layout: releases
+toc: true
+last_modified_at: 2022-01-27T22:07:00+08:00
+---
+# [Release 0.11.0](https://github.com/apache/hudi/releases/tag/release-0.11.0) ([docs](/docs/quick-start-guide))
+
+## Release Highlights
+
+### Multi-Modal Index
+
+In 0.11.0, we enable the [metadata table](/docs/metadata) with synchronous updates and
+metadata-table-based file listing by default with Spark engine, to improve the performance of partition and file listing
+on large Hudi tables. The metadata table and related file listing functionality can still be turned off by setting
+`hoodie.metadata.enable` to `false`. Please follow the upgrade steps in the migration guide to safely use this feature.
+
+We introduce a multi-modal index in metadata table to drastically improve the lookup performance in file index and query
+latency with data skipping. Two new indices are added to the metadata table
+
+1. bloom filter index containing the file-level bloom filter to facilitate bloom filter key lookup and file pruning in
+   the writer
+2. column stats index containing the statistics of all/interested columns to improve file pruning based on key and
+   column value range in both the writer and the reader, in query planning in Spark for example.
+
+They are disabled by default. You can enable them by setting `hoodie.metadata.index.bloom.filter.enable` 
+and `hoodie.metadata.index.column.stats.enable` to `true`, respectively.
+
+### Data Skipping with Metadata Table
+
+With the added support for Column Statistics in Metadata Table, Data Skipping is now relying on the Metadata Table's
+Column Stats Index (CSI) instead of its own bespoke index implementation (comparing to Spatial Curves added in 0.10.0),
+allowing to leverage Data Skipping for all datasets regardless of whether they execute layout optimization procedures (
+like clustering) or not. To enable Data Skipping make sure to set `hoodie.enable.data.skipping` to `true`, as well as
```

Comment:
should we call out that this has to be done on the reader side? if not, some users may assume setting it on writer configs would suffice?